myProject/backend/README.md

```
@@ -0,0 +1,158 @@
+- **Daily Character Game** - Unlock random Rick & Morty characters (10 coins)
+- **Memory Card Game** - Match character pairs for rewards (5 coins entry)
+- **Daily Bonus System** - Free coins every 24 hours
+- **Character Collection** with rarity system (Common → Rare → Epic → Legendary)
+- **Coin Economy** with transaction tracking
+
+<img src="./images/readme/game-flow.png" alt="Game Flow Diagram" width="500">
+
+## 📡 API Endpoints
+
+### Authentication
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)